kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -419,11 +419,7 @@
 
             id = childrenResults.id;
 
-            const redirected = _handleTopicRedirect(route, children, id, skipped);
-
-            if (redirected) {
-              return;
-            }
+            _handleTopicRedirect(route, children, id, skipped);
```

Comment:
I don't understand this change. Well, I understand it in a literal sense but I don't know really why it's happening or where it's needed.

Reply:
I had originally added this to prevent calling `next` twice when I was trying to do all the redirect handling using `next` - I forgot to remove it when I was reverting that change, so I cleaned it up here rather than faff around more with the commit history (even more than I already have).